framework/include/outputs/AdvancedOutputUtils.h

```diff
@@ -156,6 +156,30 @@ class OutputDataWarehouse : public OutputMapWrapper<OutputData>
    * Populate the OutputData structures for all output types that are 'variable' based
    */
   OutputDataWarehouse();
+
+  /**
+   * False when the show lists for all variables is empty.
+   *
+   * When false everything should output.
+   * @see AdvancedOutput::initOutputList
+   */
+  bool hasShowList(){ return _has_show_list; }
```

Comment:
Minor comment: Generally I think we put spaces between the parans and braces when doing one liners.